lib/tmdbw.py

```diff
@@ -33,12 +33,22 @@ def __init__(self, api_key):
         self.genres = json.loads(self._request("/genre/movie/list", {"language": "en-US"}))
         self.images_base_url = self.config["images"]["base_url"]
 
-    def _request(self, request, params={}):
-        params = dict(self.api_params.items() + params.items())
-        return requests.get(self.service_url + request, params).content
+    def _request(self, request, params={}, base=None):
+        if base is None:
+            base = self.service_url
+
+        while True:
+            params = dict(self.api_params.items() + params.items())
+            req = requests.get(base + request, params)
+            if req.status_code == 429:
+                retry_after = req.headers["Retry-After"]
```

Comment:
Is the "Retry-After" header always an integer (seconds)? MDN reports it can also be [dates](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Retry-After). I guess in this case it doesn't matter too much anyway!